### PR TITLE
Carrots now properly contain Oculine

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Eating carrots directly now heal your eyes and you can juice them for raw oculine at a slightly lower rate than eating them raw. Also, carrot juice now injects 0.5u of oculine as this is part of the missing 1u from grinding them.

Solves ##24227.

**Changelog**

:cl: Ubaser
- fix: Carrots now properly contain Oculine, and you can Juice them to obtain the raw chemical.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -378,12 +378,14 @@
...
       food:
-        maxVol: 9
+        maxVol: 12
         reagents:
         - ReagentId: JuiceCarrot
           Quantity: 5
         - ReagentId: Vitamin
           Quantity: 4
+        - ReagentId: Oculine
+          Qauntity: 3
```
Comment (DrMelon) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:388]: Should be "Quantity", and not "Qauntity".

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 2]
```diff
@@ -393,6 +395,8 @@
       reagents:
       - ReagentId: JuiceCarrot
         Quantity: 10
+      - ReagentId: Oculine
+        Qauntity: 2
```
Comment (DrMelon) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:399]: "Quantity" vs "Qauntity" again.

Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:399]: LOL

**Comment by DTSpawn** [comment]
Shouldn't Oculine be added is seeds.yml file?

**Comment by GreyMario** [comment]
actually, yes

**Comment by UbaserB** [comment]
wut

**Comment by GreyMario** [comment]
The produce item definition itself is actually irrelevant to adding chems to produce generated from growing and harvesting plants. You'll need to add the chemical to the carrot entry in seeds.yml.

**Comment by UbaserB** [comment]
> The produce item definition itself is actually irrelevant to adding chems to produce generated from growing and harvesting plants. You'll need to add the chemical to the carrot entry in seeds.yml.

oh i didnt know it worked that way, im going to make a hotfix soon.

**Pull request merged**